src/ofdm.c

```diff
@@ -1660,14 +1677,25 @@ void ofdm_get_demod_stats(struct OFDM *ofdm, struct MODEM_STATS *stats) {
     assert(ofdm_rowsperframe < MODEM_STATS_NR_MAX);
     stats->nr = ofdm_rowsperframe;
 
-    for (c = 0; c < ofdm_nc; c++) {
+    /* Just copy data frames */
+
+    for (c = 1; c < (ofdm_nc + 1); c++) {
         for (r = 0; r < ofdm_rowsperframe; r++) {
```

Comment:
@srsampson - around line 1390 of ofdm.c:
 ```
          ofdm->rx_np[(rr * ofdm_nc) + (i - 1)] = rx_corr;
```
so I think the original rx_np index might have already accounted for the pilots?  Be good to have some way to check, like linking with freedv-gui and seeing if the scatter diagram works.

Reply:
You are correct. My eyes no longer saw "np".